Mathlib/RingTheory/RootsOfUnity/AlgebraicallyClosed.lean

```diff
@@ -0,0 +1,37 @@
+/-
+Copyright (c) 2024 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.RingTheory.RootsOfUnity.EnoughRootsOfUnity
+import Mathlib.NumberTheory.Cyclotomic.Basic
+
+/-!
+# Instances for HasEnoughRootsOfUnity
```

Comment:
```suggestion
# Algebraically closed fields have enough roots of unity
```

Reply:
I'd rather leave the title in view of eventually adding an instance for `Circle` as well.